qt-bugs@ issue : none yet
bugs.kde.org number : none
applied: no
author: Dirk Mueller <mueller@kde.org>

This fixes Qt buildkey generation to use the same buildkey for all gcc 4.0.x compilers, because
they produce binary compatible code. This way one doesn't have to rebuild all of Qt and all
Qt plugins just because of the daily gcc version bump.

--- configure.orig	2005-07-29 17:04:29.000000000 +0200
+++ configure	2005-07-29 17:04:54.000000000 +0200
@@ -3030,6 +3030,9 @@
     *3.*)
 	COMPILER_VERSION="3.*"
 	;;
+    *4.0*)
+	COMPILER_VERSION="4.0*"
+	;;
     *)
 	;;
     esac
